patterns/2-structured/maturity-model.md

```diff
@@ -0,0 +1,231 @@
+
+**Decisions**
+
+In order to motivate colleagues to contribute work outside of their core team they need visibility into the decision making process of the host project - but also feel that their voices are being heard and valued.
+
+* DC-0: Decision-makers often intentionally or accidentally withhold data and resources related to project decisions.
+* DC-1: Materials that are part of decision-making practices become available for review after decisions are finalized.
+* DC-2:	People feel like they know about—and are helping to shape—most (but not all) important decisions as those decisions are unfolding. Materials that are part of decision-making practices are available at defined project milestones
+* DC-3: People feel like they are a part of a shared, standard process for collective decision-making that the organization endorses. Materials that are part of decision-making practices are continuously accessible during work processes.
+
+**Helpful Resources**
+
+In order to attract contributors helpful supporing material needs to be easily accessible.
+
+* RS-0: Individuals and teams neither contribute to nor draw upon a shared repository of knowledge.
+* RS-1:	Individuals and teams release project materials for review internally, after they've finished their work. Individuals and teams share resources, but in disconnected, fragmented, or individualized/siloed systems or repositories. Individuals and teams share resources, but there is no commonly expressed or shared understanding of the criteria used to determine whether information is sensitive or not. Do not "share thinking on others".
+* RS-2: Individuals and teams make project-related materials accessible to some members of project teams according to clearly defined and shared formats and/or protocols. Individuals and teams withhold sensitive data and resources, provide limited details, context, and scope.
+* RS-3: Individuals and teams make project-related materials broadly accessible to the organization—and possibly outside the organization as well—according to clearly defined and shared formats and/or protocols. Individuals and teams who must withold sensitive data and resources are clear about what they're not sharing, and others understand why those materials are not available to them.
+
+**Stories**
+
+When working in host teams mistakes will automatically be widely visible. In order keep contribution levels up, corporate culture needs to celebrate failure as an oportunity for growth and learning.
+
+* ST-0: Individuals and teams don't share neither successes nor failures for others to learn from.
+* ST-1: Individuals and teams are comfortable sharing stories about successes, but not about failures.
+* ST-23	Individuals and teams are comfortable sharing stories of successes and failures during retrospectives and reviews.
```

Comment:
Did you mean step 2 here instead of 23?

```suggestion
* ST-2: Individuals and teams are comfortable sharing stories of successes and failures during retrospectives and reviews.
```